packages/web-components/src/image/image.template.ts

```diff
@@ -0,0 +1,23 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { Image } from './image.js';
+
+/**
+ * Template for the Image component
+ * @public
+ */
+export const template: ElementViewTemplate<Image> = html<Image>`
+  <template>
+    <img
+      alt="${x => x.alt}"
+      block="${x => x.block}"
+      border="${x => x.border}"
+      borderRadius="${x => x.borderRadius}"
+      fit="${x => x.fit}"
+      margin="${x => x.margin}"
+      role="${x => (x.presentation ? 'presentation' : null)}"
+      shadow="${x => x.shadow}"
+      shape="${x => x.shape}"
+      src="${x => x.src}"
+    />
```

Comment:
A few thoughts here - 

- This shadows the element and makes it a requirement for us to enumerate _any_ attribute that needs to go *onto* this image. So, if we need to decorate this with `aria-`, we need to enumerate and create an attribute and corresponding property for every valid aria attribute. This is the same for any valid attribute for a native image, so we need to enumerate the lazy loading attributes as well.
- 1 above is super costly and a lot of JS to add a simple image
- I think a more performant and ergonomic solution for developers here is to let the platform do the work. The intent of the image component from what I can tell is to provide "art direction" and visual styling to an image (border, radius, block, shadow, etc...). We should keep this as low cost as possible IMO and mapping towards something like text would give us more benefit. 
Consider this approach:
```html
<fluent-image border shape="circle"><img src="/foo" alt="foo image" /></fluent-image>
```
This allows us to do *both* - we can provide the intent of the component and provide encapsulation for the visual styles for the image, while leveraging _more_ of the platform and not having to enumerate upwards of twenty or so attributes just to render an image.